Core/src/org/sleuthkit/autopsy/discovery/search/SearchFiltering.java

```diff
@@ -116,6 +116,9 @@
             resultList.add(new ResultFile(abstractFile));
         }
 
+        //manually add the attributes for filters which use alternate non filters and could be used by grouping or sorting
```

Comment:
We are currently adding the grouping and sorting attributes right before we group and sort in FileSearch.runFileSearch so my comment was very misleading by mentioning that it could also be used by grouping and sorting.

I can pass along the additional arguments for grouping and sorting to this SearchFiltering.runQueries and then to this method getResultList and move the adding of grouping and sorting attributes into this method to re-order in a similar fashion to your fix for the Domain Search.

Thank you.

Reply:
Or wait need to think about this, I can't remember what moving it will get me.

Reply:
Yeah didn't need to be moved apparently not initializing the frequency variable to unknown for file results was basically the entire problem with file based other occurrences.